Session/Notifications/Types/AppNotificationCategory.swift

```diff
@@ -8,22 +8,23 @@ enum AppNotificationCategory: CaseIterable {
     case incomingMessage
     case errorMessage
     case threadlessErrorMessage
+    case info
 }
 
 extension AppNotificationCategory {
     var identifier: String {
         switch self {
-            case .incomingMessage: return "Signal.AppNotificationCategory.incomingMessage"
-            case .errorMessage: return "Signal.AppNotificationCategory.errorMessage"
-            case .threadlessErrorMessage: return "Signal.AppNotificationCategory.threadlessErrorMessage"
+            case .incomingMessage: return "Session.AppNotificationCategory.incomingMessage"
+            case .errorMessage: return "Session.AppNotificationCategory.errorMessage"
+            case .threadlessErrorMessage: return "Session.AppNotificationCategory.threadlessErrorMessage"
+            case .info: return " Session.AppNotificationCategory.info"
```

Comment:
I'm not sure it's a huge issue even if this is the case (considering it would only be fore notifications which existed during the "transition" to the update) but if we change these identifiers and someone had a notification from before the update I assume it would fail to get the correct category?

Reply:
This can be a problem, I can keep the old identifiers for backwards compatibility in this PR and totally change it in future release.